osu.Game.Tests/Visual/Gameplay/TestScenePlayerLoader.cs

```diff
@@ -353,6 +353,22 @@ public void TestEpilepsyWarningEarlyExit()
             AddUntilStep("sound volume restored", () => Beatmap.Value.Track.AggregateVolume.Value == 1);
         }
 
+        [Test]
+        public void TestQuickRetry()
+        {
+            AddStep("load dummy beatmap", () => resetPlayer(false));
+            AddUntilStep("wait for current", () => player.IsCurrentScreen());
+
+            AddStep("Restart map normally", () => player.Restart());
+            AddUntilStep("wait for current", () => player.IsCurrentScreen());
+
+            AddStep("Restart map with quick retry hotkey", () =>
+            {
+                InputManager.UseParentInput = true;
+                InputManager.PressKey(Key.Tilde);
+            });
+        }
+
```

Comment:
Not sure if it's fine to leave this in the `PlayerLoader` test scene. I don't think it is worth to create a new test scene just for this one test case.

Reply:
I mean the placement is fine, but it's not actually testing anything. I'll fix it.